### PR TITLE
As/2734 base fee based calculation

### PR DESCRIPTION
**Pull request by adamskrodzki**
Change pipelines in a way described in 
https://app.shortcut.com/oazo-apps/story/2734/change-gasprice-estimator-to-use-basefee-and-set-max-priority-fee-per-gas-to-5-gwei

### CODE REVIEW

File: blockchain/prices.ts [hunk 1]
```diff
@@ -18,16 +18,41 @@ export interface Ticker {
   [label: string]: BigNumber
 }
 
-export type GasPrice$ = Observable<BigNumber>
+export type GasPriceParams = {
+  MaxFeePerGas: BigNumber
```
Comment (pacxiu) [inline blockchain/prices.ts:22]: Too keep naming consistent on object params across app it should start from lowercase so:
```
maxFeePerGas: BigNumber
maxPriorityFeePerGas: BigNumber
```

also small typo: `Pririty`

File: blockchain/prices.ts [hunk 2]
```diff
@@ -18,16 +18,41 @@ export interface Ticker {
...
+      const retVal = {
+        MaxFeePerGas: new BigNumber(block.baseFeePerGas).multipliedBy(2).plus(minersTip),
+        MaxPrirityFeePerGas: minersTip,
+      } as GasPriceParams
+      console.log(
+        `GasPriceParams = { MaxFeePerGas:${retVal.MaxFeePerGas}, MaxPrirityFeePerGas:${retVal.MaxPrirityFeePerGas} }`,
+      )
+      return retVal
+    }),
+    distinctUntilChanged(
```
Comment (pacxiu) [inline blockchain/prices.ts:52]: I think you can use `isEqual` from lodash, so it will be just `distinctUntilChanged(isEqual)`

**Comment by pacxiu** [comment]
I think there will also need to be some changes inside `doGasEstimation` function to calculate estimated fees as it also uses `gasPrice$`

I guess TS haven't caught that as it fails on Prettier from what I see

File: blockchain/prices.ts (hunk 2 above)
Comment (adamskrodzki) [reply blockchain/prices.ts:52]: Actually there are, in 
helpers/form.ts

**Review by pacxiu — APPROVED** [review]

**Pull request merged**